packages/cli/src/index.ts

```diff
@@ -690,7 +690,8 @@ const main = async () => {
     if (shouldCollectMetrics) {
       const category = 'Command Invocation';
 
-      metric
+      if (!metric) metric = metrics();
+      metrics()
```

Comment:
```suggestion
      metric
```

This should reuse the reference that was just created, right?